test/tilt_livescripttemplate_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'test_helper'
+require 'tilt'
+
+begin
+  require 'tilt/livescript'
+
+  module LiveScriptTests
+    def self.included(mod)
+      mod.class_eval do
```

Comment:
Unless you want to include LiveScriptTests in multiple test classes, you might as well write the code below directly into the test class:

```
class LiveScriptTemplateTest < Minitest::Test
  setup do
    …
  end

  test "bare is disabled by default" do
    …
  end
```